### PR TITLE
Update Filter Complex to include changed to the Row Filter

### PR DESCRIPTION
**Pull request by wongpratan**

### CODE REVIEW

File: AppBuilder/platform/views/viewProperties/ABViewPropertyFilterData.js [hunk 1]
```diff
@@ -1009,44 +1026,41 @@ export default class ABViewPropertyFilterData extends ABViewProperty {
...
+         // this.filter_popup = webix.ui({
+         //    view: "popup",
+         //    id: ids.component,
+         //    width: 800,
+         //    hidden: true,
+         //    body: this.rowFilter.ui,
+         // });
 
          // register callbacks:
          for (var c in logic.callbacks) {
```
Comment (Hiro-Nakamura) [inline AppBuilder/platform/views/viewProperties/ABViewPropertyFilterData.js:1038]: do we still support callbacks?

File: index.js [hunk 2]
```diff
@@ -31,8 +31,10 @@ import ganttCSS from "./js/webix/components/gantt/gantt.css";
 
 import kanban from "./js/webix/components/kanban/kanban.js";
 import pivot from "./js/webix/components/pivot/pivot.js";
-import querybuilder from "./js/webix/components/querybuilder/querybuilder.js";
 import report from "./js/webix/components/reports/reports.js";
+// import querybuilder from "./js/webix/components/querybuilder/querybuilder.js";
```
Comment (Hiro-Nakamura) [inline index.js:35]: if you want the `querybuilder` to be global then you might have to : 
```
const querybuilder = require("./path/here");
window.querybuilder = querybuilder;
```

The current `index.js` has some examples with the `.gantt` and `.reports`

File: AppBuilder/platform/FilterComplex.js [hunk 3]
```diff
@@ -307,149 +317,294 @@ module.exports = class FilterComplex extends FilterComplexCore {
...
-         rule == "not_in_data_collection"
-      ) {
-         this.uiInDataCollectionValue($valueElem, field);
-      } else if (field.key == "list") {
-         this.uiListValue($valueElem, field);
+   // HACK: have to overwrite Webix Query's function to support our custom input requirement.
+   // HooWoo
+   uiQueryCustomValue() {
+      // Could not require on the top of the page. (Webix cound not found error). Yahoo
+      const Query = require("../../js/webix/components/query/query.js");
```
Comment (Hiro-Nakamura) [inline AppBuilder/platform/FilterComplex.js:335]: Checkout my comment on the `index.js` file and see if that will work instead of this.

**Review by Hiro-Nakamura — CHANGES_REQUESTED** [review]
Almost there!  I can't wait to have this in the repo!

File: AppBuilder/platform/views/viewProperties/ABViewPropertyFilterData.js (hunk 1 above)
Comment (wongpratan) [reply AppBuilder/platform/views/viewProperties/ABViewPropertyFilterData.js:1038]: There are .change and .save events like EventEmitter.

I don't sure do we still need to callback style ?

**Pull request merged**